src/components/App/MainMenu/SessionMenuItem.tsx

```diff
@@ -0,0 +1,69 @@
+import { signIn, signOut, useSession } from "next-auth/react";
```

Comment:
I really really love how this section turned out. It looks amazing. Great work 🫶 

![A man, probably from South-America, sitting in front of the camera, saying "Mucho mucho joy"](https://media1.tenor.com/m/Qe4HyPPaoFAAAAAd/ted-lasso-tedlassogifs.gif)